### ISSUE TITLE
catalytic activity, acting on DNA | IEA with ARBA00027986 

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-annotation#4782]
* **ARBA rule (ID and label):**

GO:0140097 | catalytic activity, acting on DNA | IEA with ARBA00027986 | GO_REF:0000117

* **Sequences with problematic annotation (ID + gene/protein name):**

psm1 https://www.pombase.org/gene/SPBC29A10.04

* **Description of issue**

Hi, can you trace where this rule comes from?
As far as I can see no psm1 ortholog has annotations in this branch (at least at the Alliance)

This is an ATPase, and it topologically entrams DNA, but there is no DNA metabolism here (the DNA is not altered). The ATPase provides energy for cohesin loading.

Interesting aside,  There are 1620 ARBA annotations for fission yeast /1647 ARBA are now  recovered by exiting annotation from some source, only 27 unique (including this one, so overall there are very few FP)

**Comment by PedroRaposo** [comment]
Hello,
The annotation comes from 5 Swiss-Prot proteins where they meet the ARBA condition IPR003395 + IPR027417 + Taphrinomycotina: https://www.uniprot.org/uniprotkb?query=%28xref%3Ainterpro-IPR003395%29+AND+%28xref%3Ainterpro-IPR027417%29+AND+%28taxonomy_id%3A451866%29&facets=reviewed%3Atrue

It seems that this 5 SP includes the protein in question, O94383. So, it seems ARBA is using its own propagation to learn from themselves - I will talk to the team about this.

**Comment by ValWood** [comment]
Actually I could be wrong, or, there could be a GO issue.


1.  Note that IPR003395 + IPR027417  should be equivalent  because ALL  IPR010935 should also have IPR027417 

2. This family includes all of the SMC subunit ATPases (from condensin, cohesion and SMC complex). 

These (at least some) are DNA translocases:
 DNA translocase activity (GO:0015616)
Definition | Parents
Generation of movement along a single- or double-stranded DNA molecule, driven by ATP hydrolysis.
catalyse *movement* along a DNA molecule, 


This  term is a descendant of 
GO:0140097 | catalytic activity, acting on DNA 
Defined 
Catalytic activity that acts to modify DNA.

but as far I'm aware DNA is not "modified" by the translocase activity? I could be wrong so I'd like to confirm this with @colinlog   (in this case the GO parent would be correct)

3. Pombe cut14	 was annotated to GO:0036292	DNA rewinding, this was incorrect and has been removed.

**Comment by pgaudet** [comment]
@ValWood It's considered a catalytic activity because it's an ATPase. 

I guess your point is that the substrate is not DNA? Do you want to remove this relation?

**Comment by ValWood** [comment]
Correct, the ATPase  isn't acting on DNA, the ATPase is acting on the cohesin

**Comment by colinlog** [comment]
Hello,
The eukaryotic ATPases of the Structural Maintenance of Chromsomosomes (SMC) work in pairs (SMC1/3 = cohesin, SMC2/4 = condensin, SMC5/6 = not named). All of the complexes that contain these SMC ATPase pairs have chromatin loop extrusion activity. They translocate DNA to enlarge DNA loops. This has been seen at the single molecule level for condesnin and cohesisn and very recently (PMID: 38295804, https://doi: 10.1016/j.molcel.2024.01.005) similar evidence was obtained for the SMC5/6 pair. 

Therefore, next to the chromosome condensation BP for condensin and the BP sister chromatid pairing for the cohesin, it looks to me that GO will need to generically annotate the SMC motors to a MF that stands for 'chromatin loop extrusion'.

**Comment by PedroRaposo** [comment]
So in other words are we saying that not all 161 ATPases (https://www.uniprot.org/uniprotkb?query=%28go_manual%3A0140097%29+AND+%28reviewed%3Atrue%29+AND+%28ec%3A3.6.4.12%29) that contain this GO term GO:0140097 is acting on DNA, and therefore these would need to be reviewed by our curators?

**Comment by ValWood** [comment]
Hi @PedroRaposo  that shouldn't be necessary.
It depends on the source of the problem annotation. Most will likely be IEA.

Most of the issues should  be fixed because the InterPro mappings would be updated, and then the rule should no longer generate annotations.
I will have requested mapping fixes for the violations that I spotted.

There is a remaining term
ATP-dependent DNA damage sensor activity (GO:0140664)
which could be problematic.
From a quick look, this term combines a DNA sensing activity , with an ATPase activity which appear to be separate activities in the MUtS family proteins.
Although since these proteins seem to have both DNA damage sensing, and some DNA metabolic activity, the resulting annotations won't be incorrect aven though the term itself is suspect.
@pgaudet do you agree?

**Comment by ValWood** [comment]
In fact for the problem I reported 
https://www.ebi.ac.uk/QuickGO/annotations?geneProductId=O94383&aspect=molecular_function
I no longer see an ARBA annotation to DNA metabolic process so it might be that this one is corrected already.

**Comment by colinlog** [comment]
The SMC proteins form (hetero)dimers that change the topology of DNA by havong multiple DNA interaction surfaces/domains. They exist in all cells to organize genomic DNA, eg; mukB in bacteria. The SMC proteins really do act on DNA. The product of the ATPase reaction are indeed SMC dimer conformational changes but these translate to novel DNA (or chromatin fibre) topologies (in archea and eukaryotes), often as anchored loops  or as plectonemic or constrained supercoils.  The SMC ATPases are considered to be a novel class of motors as they catalyse DNA/chromatin movements.

A second class of non SMC-type nucleic acid translocases are the  DEAD/H nucleic-acid dependent ATPases. Many are DNA or RNA helicases that translocate along the DNA/RNA polymers. Some, called the SNF2-types act on nucleosomes,  or on other DNA-bound proteins (eg; MOT1 on DNA-bound TBP) and can have as additional nucleosome-related activities such as the ATP-dependent exchange of H2A-H2B dimers including H2A.Z or H2A.X variants. One actually translocates 'hexamers' consiting of a H3(2)-H4(2) tetramer and only one H2A-H2B variant. These SNF2-type translocases/motors often have protein dependencies to express their 'nucleic acid-stimulated ATPase activity such as (post translationally modified) histone tails.

Hope this helps you resolve the issue.

**Comment by colinlog** [comment]
I think the trick is to try to flip the perspective. It is as much the protein that translocates along the DNA as the DNA that is being translocated by a protein. In the latter perspective the product of the reaction is a new DNA topology. That is the molecular function of SMC dimers. Not the fact that they tranlocate themselves along the DNA but rather tranlocating the DNA/chromatin. This usually involves changes in supercoiling. For SMCs it is usually positive supercoiling that is involved, which could be viewed as 'overwinding' and that translates in 'free plectonemes' or protein-constrained loops.

Furthermore, as the SMC dimers have multiple DNA interaction surfaces, they give DNA new (supercoiled) shapes and when they change position, they also change the position of the DNA they are associated with. In that sense they are dynamic structural constitutents of chromosomes. 

There is still much to be understood on the SMC1/3, 2/4 and 5/6 heterodimers MF and the associated BPs. The bacterial proteins seem to be better understood.

**Comment by ValWood** [comment]
but are these topological changes rather than catalytic activities? It might be OK, but traditionally 

DNA topological change
https://www.ebi.ac.uk/QuickGO/term/GO:0006265
is not defined as a "catalytic activity" 
so this would need to be reviewed.

I think they way GO is organized at the moment, the unwinding which involves strand separation  (helicase) is classed as 'catalytic activity' but  topological changes are not classed as catalytic activity. 

We might need to review these?

**Comment by colinlog** [comment]
catalysis is the capacity of a protein (complex) to repeatedly perform a
biochemical reaction without being changed itself. The SMC motors are not
different from other enzymes in this respect. Although, I understand that
it is a little nebulous to have a 'new DNA/chromatin fibre topology' as
reaction product. It would be some type of isomerisation, I believe rather
than a novel covalent chemical reaction product.

**Comment by ValWood** [comment]
but isn't the reaction in these cases ATP hydrolysis?

**Comment by ValWood** [comment]
Perhaps we need to revise. the definition of 
GO:0140097  catalytic activity, acting on DNA
Definition ([GO:0140097 GONUTS page](http://gowiki.tamu.edu/wiki/index.php/Category:GO:0140097))
Catalytic activity that acts to **modify DNA**.

if we want this to also cover topological changes that do not 'metabolise' DNA?

My understanding of this term
Catalytic activity that acts to modify DNA.
is that the DNA needed to be chemical altered (which might be the case, but this isn't obvious to me). Maybe I am overthinking it!

Also if I annotate the cohesin subunits to "DNA translocase"
based on the current parents, this would be correct GO:0008094   
ATP-dependent activity, acting on DNA

So I guess we can close this. It isn't an ARBA problem.

**Comment by pgaudet** [comment]
Right, this is broader, I'll put this on the next ontology editors call.

**Comment by ValWood** [comment]
Update For the record, the sensing activity is ATP dependent
https://pubmed.ncbi.nlm.nih.gov/33302090/
